apis/kueue/v1beta1/workload_types.go

```diff
@@ -578,6 +578,14 @@ const (
 	// WorkloadMaximumExecutionTimeExceeded indicates that the workload exceeded its
 	// maximum execution time.
 	WorkloadMaximumExecutionTimeExceeded = "MaximumExecutionTimeExceeded"
+
+	// WorkloadWaitForPodsStart indicates that the PodsReady condition is set before
+	// the workload has started
```

Comment:
```suggestion
	// WorkloadWaitForPodsStart indicates the reason for PodsReady=False condition
	// when the pods have not been ready since admission, or the workload is not admitted.
```
I think we describe the status of the Workload, not when the condition was added.